frontend/src/components/Register.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector, batch } from 'react-redux'
+import { useNavigate } from 'react-router-dom'
+import { Buttons } from '../GlobalStyles'
+import { InnerWrapper } from '../GlobalStyles';
+import { OuterWrapper } from '../GlobalStyles';
+import { Batman } from '../GlobalStyles';
```

Comment:
What about instead using comma to import all styles from that file, as you did on row 8? :)